### DISCUSSION TITLE
Quickstart osmt-app-1 not starting

### DISCUSSION THREAD

**Discussion by rudy514** [wgu-opensource/osmt#124] [Asking for Help] [answered]
When building OSMT from the quickstart configuration using the develop branch, osmt-app-1 does not start with the error 

"standard_init_linux.go:228: exec user process caused: no such file or directory"

As a note, we are using docker on windows if that makes any difference. 

![image](https://user-images.githubusercontent.com/40806925/150213977-2c211f5a-845e-4772-8525-19a287f40fc3.png)

**Comment by JohnKallies** [comment, marked as answer]
Hello Rudy. I don't have much experience with Docker on Windows...

Can you please look at the docker logs for the app container, and share anything interesting you find? I'm thinking additional context for whatever file or directory it's looking for.
